rest/utilities_testing.go

```diff
@@ -46,7 +46,12 @@ type RestTester struct {
 
 func (rt *RestTester) Bucket() base.Bucket {
 
-	if rt.RestTesterBucket == nil {
+	if rt.RestTesterBucket != nil {
+		return rt.RestTesterBucket
+	}
+
+	// Put this in a loop in case certain operations fail, like waiting for GSI indexes to be empty
+	for {
```

Comment:
This looks like unlimited retry in the case where WaitForIndexEmpty fails (after 60s worth of retries) and we then drop all the indexes.  Do we actually want to drop indexes and retry more than once?

Reply:
I limited the number of retries to 2 -- once before drop indexes, and once after.  If it still can't complete the operation, it panics (test code only)